src/transformers/Streamable.js

```diff
@@ -0,0 +1,40 @@
+import { URL } from 'url';
+import fetch from 'node-fetch';
+
+export const getNormalizedStreamableUrl = url => {
+  const splitUrlSegments = url.split('/');
+  return splitUrlSegments.length > 4
+    ? splitUrlSegments[4]
+    : splitUrlSegments[3];
+};
+
+const getTrimmedPathName = pathname => pathname.replace(/^\/|\/+$/g, '');
+
+export const shouldTransform = url => {
+  const { host, pathname } = new URL(url);
+  const trimmedPathName = getTrimmedPathName(pathname).split('/');
+
+  return (
+    ['streamable.com', 'www.streamable.com'].includes(host) &&
+    trimmedPathName.length > 0 &&
+    trimmedPathName.length <= 3 &&
+    ((trimmedPathName.length === 1 &&
+      ![
```

Comment:
Maybe also extract the array to a `ignoredPaths` variable, like we have `podcastTypes` in the Spotify transformer